src/main/ast-to-doc.js

```diff
@@ -13,6 +13,9 @@ const addAlignmentToDoc = docBuilders.addAlignmentToDoc;
 const docUtils = doc.utils;
 
 function printAstToDoc(ast, options, addAlignmentSize) {
+  options = Object.assign({}, options, {
+    originalText: options.originalText || ""
+  });
```

Comment:
You can make this shorter (& clearer IMO) by doing

```js
options = Object.assign({ originalText: "" }, options)
```

Reply:
Ah, of course. Fixed in https://github.com/prettier/prettier/pull/4677/commits/752b71dceb32b8c3ac2482750cc8cc656a583731